game/menu/gamemanager.gd

```diff
@@ -13,22 +20,169 @@ func _ready():
 
 
 func load_game(game_cfg: ConfigFile):
+	_last_loaded_game = game_cfg.get_meta("folder_name")
+	_started_playing_game = OS.get_ticks_msec()
 	# load the games main scene
-	var scene = load(game_cfg.get_meta("folder_path") + game_cfg.get_value("game", "main_scene"))
-
+	var scene = load(
+		(
+			"res://games/"
+			+ game_cfg.get_meta("folder_name")
+			+ "/"
+			+ game_cfg.get_value("game", "main_scene")
+		)
+	)
 	var err := get_tree().change_scene_to(scene)
 	if err != OK:
 		prints("Error", err)
 		return
 	_main.hide()
 
 
+func _save_data(file_name: String, data, game: String) -> int:
+	"""Save data to a file with for the given game.
```

Comment:
I don't like the multiline comments, they look like strings. It's a preference i think.

Reply:
They are strings

Reply:
I know it's for the latest godot version, they use ‘##’ for description.
https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_documentation_comments.html
But i think it”s a good guideline to use

Reply:
I did not know this either :P